addons/project/views/project_views.xml

```diff
@@ -1339,24 +1327,30 @@
                                     <div class="o_kanban_record_headings">
                                         <strong class="o_kanban_record_title">
-                                            <s t-if="!record.active.raw_value"><field name="name" widget="name_with_subtask_count"/></s>
-                                            <t t-else=""><field name="name" widget="name_with_subtask_count"/></t>
+                                            <s t-if="!record.active.raw_value"><field name="name"/></s>
+                                            <t t-else=""><field name="name"/></t>
                                         </strong>
-                                        <span invisible="context.get('default_project_id', False)"><br/><field name="project_id" widget="project_private_task" options="{'no_open': True}"/></span>
-                                        <span t-if="record.allow_milestones.raw_value and record.milestone_id.raw_value" t-attf-class="{{record.has_late_and_unreached_milestone.raw_value ? 'text-danger' : ''}}">
+                                        <span invisible="context.get('default_parent_id', False)">
+                                            <t t-if="record.parent_id.raw_value">
+                                                <br/><field name="parent_id"/>
+                                            </t>
+                                        </span>
+                                        <span invisible="context.get('default_project_id', False)">
                                             <br/>
-                                            <field name="milestone_id" options="{'no_open': True}" />
+                                            <field name="project_id" widget="project_private_task" options="{'no_open': True}"/>
                                         </span>
-                                        <br />
                                         <t t-if="record.partner_id.value">
-                                            <field name="partner_id"/>
+                                            <field name="partner_id" class="text-muted" style="display: block; margin-top: 4px;"/>
                                         </t>
+                                        <span t-if="record.allow_milestones.raw_value and record.milestone_id.raw_value" t-attf-class="{{record.has_late_and_unreached_milestone.raw_value ? 'text-danger' : 'text-muted'}}" style="display: block; margin-top: 4px;">
+                                            <field name="milestone_id" options="{'no_open': True}" />
+                                        </span>
                                     </div>
                                 </div>
-                                <div class="o_kanban_record_body">
+                                <div class="o_kanban_record_body" t-attf-class="{{record.is_closed.raw_value ? 'opacity-50' : 'text-muted'}}">
```

Comment:
```suggestion
                                <div class="o_kanban_record_body" t-att-class="record.is_closed.raw_value ? 'opacity-50' : 'text-muted'">
```
When your whole t-attf is inside curly brackets, you can use t-att and no brackets.
In t-attf, we evaluate what's inside double curly brackets ; in t-att, we evaluate everything.

Reply:
There are other occurrences of this in this PR :slightly_smiling_face: